config/core/configmaps/features.yaml

```
@@ -149,3 +149,16 @@ data:
     # 1. Enabled: enabling init containers support
     # 2. Disabled: disabling init containers support
     kubernetes.podspec-init-containers: "disabled"
+
+    # Controls whether persistent volumes support is enabled or not.
+    # 1. Enabled: enabling persistent volumes support
+    # 2. Disabled: disabling persistent volumes support
+    kubernetes.podspec-persistent-volumes-claims: "disabled"
```

```suggestion
    kubernetes.podspec-persistent-volume-claims: "disabled"
```

think this is singular in the spec

---

.. though then it wouldn't match `podspec-persistent-volumes-write` so maybe your way is better even if inconsistent. hm :(.

---

Yeah I thought too about the naming inconsistency, so you mentioned the K8s spec uses `PersistentVolumeClaimSpec` no plural anyway. I tried to make it similar so the other option would be: `podspec-persistent-volume-writes`, have plural at the last part only. Wdyth?

---

I switched to:
```
 kubernetes.podspec-persistent-volume-claim
 kubernetes.podspec-persistent-volume-write
```